09-intermediate-rails/oauth.md

```diff
@@ -26,74 +29,65 @@ gem "omniauth"
 
-To use a `.env` with Rails, head back to the Gemfile. Add `gem 'dotenv-rails'` to the `:development` group in the Gemfile:
+To use a `.env` file with Rails, you **must** do all of these steps to gain access to the data you set in the `.env` file.
+1. Add `gem 'dotenv-rails'` to the `:development` group in the Gemfile  
 
-```ruby
-#Gemfile
-group :development, :test do
-  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
-  gem 'byebug'
-  gem 'dotenv-rails'
-end
-```
+    __QUESTION:__ Why add it to the `:development` group? Why does the group matter?
 
-Then head to the terminal for Yet Another Bundle® (`$ bundle`).
+1. Install the gem using `bundle`
+1. Add the `.env` file to our `.gitignore`  
 
-__QUESTION:__ Why add it to the `:development` group? Why's it matter?
+    This will help prevent us from accidentally publishing it on GitHub. Open up `.gitignore` in the project root, and add these lines:
 
-Next, before we create the file we will add it to our `.gitignore`. This will help prevent us from accidentally publishing it on GitHub. Open up `.gitignore` in the project root, and add these lines:
+1. Create the `.env` file in the root directory with `$ touch .env`.
 
-```
-# Ignore the .env file (it's full of secrets!)
-.env
-```
-
-Now create the `.env` file with `$ touch .env`. This file is a collection of key/value pairs; add the application credentials from Github like this:
+  This file is a collection of key/value pairs. We will add the application credentials from GitHub like this:
 
-```bash
-GITHUB_CLIENT_ID: fd6XXXXXXXX
-GITHUB_CLIENT_SECRET: y6wXXXXXXX
-```
+  ```bash
+  GITHUB_CLIENT_ID: fd6XXXXXXXX
+  GITHUB_CLIENT_SECRET: y6wXXXXXXX
+  ```
 
-With that done, the Github application credentials will now be available to Rails via the `ENV` constant.
+**With that done, the GitHub application credentials will now be available to Rails via the `ENV` constant.**
 
 #### Accessing Credentials
-Now that you have application credentials, let's configure Rails to use them. To do this, create a new _initializer file_. Initializers are files that run as part of the Rails boot process. Initializers go in the `config/initializers/` directory. From the terminal, create a new initializer with `$ touch config/initializers/omniauth.rb`. Open this file and add the following code:
+Now that you have application credentials, let's configure Rails to use them. To do this, create a new _initializer file_. **Initializers are files that run as part of the Rails start-up process**. Initializers go in the `config/initializers/` directory. From the terminal, create a new initializer with `$ touch config/initializers/omniauth.rb`. Open this file and add the following code:
```

Comment:
Might be worth mentioning that when you change an initializer, you have to restart the Rails server.